BatteryEvaluation/CurrentMonitoring.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BatteryEvaluation
+{
+    public class CurrentMonitoring
+    {
+        public bool ValidateReadingList(List<int> readingList)
+        {
+            if ((readingList != null) && (readingList.Count != 0))
```

Comment:
separate null check from empty ..and parameterize functions to validateReadingList()

if function return true better function name begin with "Is*****"

Reply:
Changed Name and separated checks 
Also Test Cases Names are changed